src/app/interactions/dialog/custom-dialog/custom-dialog.component.html

```diff
@@ -0,0 +1,18 @@
+<h4 class="sample-title">Custom Title and Actions</h4>
+<p class="sample-description">Dialog Title and Actions are re-templated.</p>
+
+<button igxButton="raised" igxRipple="white" (click)="dialog.open()">Show Dialog</button>
+<igx-dialog #dialog [closeOnOutsideSelect]="true" message="This will create a new social media account.">
+    <igx-dialog-title>
+        <div class="dialog-container">
+            <igx-icon>account_box</igx-icon>
+            <div class="dialog-title">Create a new account?</div>
+        </div>
+    </igx-dialog-title>
+    <igx-dialog-actions>
```

Comment:
Could you please change this element to `div` with `igxDialogActions` directive applied to demonstrate both approaches.